README.md

```diff
@@ -278,6 +278,9 @@ end
 
 Specs created by the generator should now have access to test helpers like `render_inline`.
 
+To use component previews, you'll want to [configure the preview path](#previewing-components) so that preview
```

Comment:
It'd be great if we could do this by default, as I believe action mailer works out of the box with rspec-rails. 

But it seems we would need to monkey patch `rspec-rails`. Take a look at https://github.com/rspec/rspec-rails/blob/d0f81dfccf92df655caeacfd1ce0ba1808779f99/lib/rspec-rails.rb#L40

Let me know what you think. We can do it in a different PR maybe, just to move this forward.

Reply:
Ah, that's interesting! Nice find. @juanmanuelramallo 

I'd vote to discuss that (and any other helpful RSpec changes we could make) in another PR if folks think what we've got here is useful enough on its own.

Reply:
Maybe we could just tell them to set the preview path to the same one but using `spec` instead of `test`:

```suggestion
To use component previews add this into your application.rb: `config.action_view_component.preview_path = "#{Rails.root}/spec/components/previews"`
```

Reply:
Good suggestion. I like being more direct to get people up-and-running with minimal fuss.